.github/workflows/call-maventests.yml

```diff
@@ -0,0 +1,17 @@
+# This workflow will build a Java project with Maven
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Maven build
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  call-workflow-maven-build:
+    uses: felleslosninger/eid-github-workflows/.github/workflows/maven-build-lib.yml@work/ID-1627
```

Comment:
Pipeline-ref mot branch?

Reply:
tek og fiksar det på ny sak, tenkte å få brukt denne saka mot branchen fyrst for å sjekke at det fungerer før den pipeline saka går til main.